extern/sector-storage/mock/mock.go

```diff
@@ -547,6 +549,11 @@ func (m mockVerifProver) VerifyAggregateSeals(aggregate proof5.AggregateSealVeri
 	return ok, nil
 }
 
+// TODO: do the thing
+func (m mockVerifProver) VerifyReplicaUpdate(update proof7.ReplicaUpdateInfo) (bool, error) {
+	return false, nil
```

Comment:
`return true, nil` should be reasonable for testing, we can do fancier things later

Reply:
Ack